CMakeLists.txt

```diff
@@ -104,6 +106,34 @@ if(NOT CUKE_DISABLE_GTEST)
 endif()
 
 #
+# Qt
+#
+
+if(NOT CUKE_DISABLE_QT)
+    set(CMAKE_PREFIX_PATH $ENV{QTDIR})
+    find_package(Qt5Core  QUIET)
```

Comment:
I wasn't sure how to do it, becouse if finding qt5 failed i'm searching for qt4 which is sufficient for the examples. REQUIRED would fail the configuration. Mabey I'll go without quiet